app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -167,12 +169,14 @@ class NavigationOverlayFragment : Fragment() {
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        ToolbarUiController(
+        toolbarUiController = ToolbarUiController(
             toolbarViewModel,
             ::exitFirefox,
             onNavigationEvent,
             serviceLocator.experimentsProvider
-        ).onCreateView(view, viewLifecycleOwner, fragmentManager!!)
+        ).apply {
+            onViewCreated(view)
```

Comment:
nit: this whole block of code should actually be in `onCreateView` rather than renaming this method: `onViewCreated` is intended for sub-classes of custom Fragments so they can react to `onCreateView` without having to override, and potentially mess up, `onCreateView`: see the javadoc of these methods for details.